go.mod

```diff
@@ -0,0 +1,72 @@
+module github.com/openshift/cluster-openshift-controller-manager-operator
```

Comment:
I want to see initial go.mod this commit is too large and makes the review hard.